gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GcsJavaClientImpl.java

```diff
@@ -0,0 +1,211 @@
+package com.google.cloud.hadoop.gcsio;
```

Comment:
copyright. I guess we missed this in the original PR.

Reply:
added